content/uc-doc/administration/nat.md

```diff
@@ -0,0 +1,59 @@
+---
+title: NAT traversal
+---
+
+[//]: # (Asterisk doc: https://wiki.asterisk.org/wiki/display/AST/Configuring+res_pjsip+to+work+through+NAT)
```

Comment:
What is suppose to do?
The preview display nothing
https://sf.production.wazo.cloud/logs/14/214/9868a328718c938bbf55fedecbdfbe1e94840a1e/check/yarn-build-copy/275ef72/public/uc-doc/administration/nat

Reply:
It's the least intrusive way to write a comment in Markdown :/
So it's normal nothing is displayed.